apps/dashboard/src/app/resources/resource-list.component.html

```diff
@@ -368,15 +368,15 @@
 
     <div
       class="pagination"
-      *ngIf="!(isEmpty | async)">
+      *ngIf="resultsLength > 0">
```

Comment:
if `isEmpty` is not used anymore, it must be removed from the component implementation

Reply:
Fixed